docs/languages/powershell.md

```diff
@@ -4,39 +4,50 @@ Area: languages
 TOCTitle: PowerShell
 ContentId: 8688bb6d-793e-4a37-aed2-5af4cfe89940
 PageTitle: PowerShell editing with Visual Studio Code
-DateApproved: 02/06/2025
+DateApproved: 02/20/2025
 MetaDescription: Learn about using PowerShell in Visual Studio Code
 ---
 # PowerShell in Visual Studio Code
 
-[PowerShell][20] is a task-based command-line shell and scripting language built on [.NET][19] that
+[PowerShell][24] is a task-based command-line shell and scripting language built on [.NET][23] that
 provides a powerful toolset for administrators on any platform.
 
-The Microsoft [PowerShell][27] extension for Visual Studio Code (VS Code) provides rich language
+The Microsoft [PowerShell][31] extension for Visual Studio Code (VS Code) provides rich language
 support and capabilities such as syntax completions, definition tracking, and linting for
-PowerShell. The extension should work anywhere VS Code itself and PowerShell Core 7.2 or higher is
-[supported][14]. For Windows PowerShell, only version 5.1 is supported and only on a best-effort
-basis. PowerShell Core 6, 7.0, and 7.1 have reached end-of-support. We test the following
-configurations:
+PowerShell. The extension should work anywhere VS Code is supported using supported versions of
+[PowerShell 7][17].
 
-- **Windows Server 2022** with Windows PowerShell 5.1 and PowerShell 7.2
-- **Windows Server 2019** with Windows PowerShell 5.1 and PowerShell 7.2
-- **macOS 11** with PowerShell Core 7.2
-- **Ubuntu 20.04** with PowerShell Core 7.2
+Support for Windows PowerShell 5.1 is provided on a best effort basis. [.NET Framework 4.8][18] or
+higher is required.
+
+[VS Code Remote Development][13] Environments, including [Github Codespaces][19] and
+[VS Code Server][14] are supported.
+
+> [!IMPORTANT]
+> [Visual Studio Code for the Web][12] is only supported for limited functionality such as basic
+> syntax highlighting, as the PowerShell engine can't run in this environment.
+
+We actively test the following configurations:
+
+- **Windows Server 2022** with Windows PowerShell 5.1 and PowerShell 7+
+- **macOS 14.7** with PowerShell 7+
+- **Ubuntu 22.04** with PowerShell 7+
```

Comment:
```suggestion
- **Ubuntu 24.04** with PowerShell 7+
```

You copied this correctly, I just thought to double check against our CI and it has indeed updated to 24.04. I'll update the repo's readme too. The rest are still accurate.